pyccel/parser/semantic.py

```diff
@@ -991,10 +991,14 @@ def _visit_IndexedElement(self, expr, **settings):
 
         return self._extract_indexed_from_var(var, args, name)
 
+    def _visit_Symbol(self, expr, **settings):
+        # While and With class are using sympy.sympify
```

Comment:
This should be fixed when you merge the master branch. I just removed this in PR #732

Reply:
Beware of the new `from sympy import Symbol` in `pyccel/ast/macros.py`

Reply:
Yes I just saw your PR .

Reply:
There are several uses of `Symbol` in that PR which need updating:
https://github.com/pyccel/pyccel/pull/732/files#diff-707dd305f5ecd064576e2cfe995707cbe5f02e1b7cf7a9d5baa6418df3a82573R1645
https://github.com/pyccel/pyccel/pull/732/files#diff-68eb50666798e9569a48a77a7d59e728348769aeea74f48daf6a5b955586170aR10
https://github.com/pyccel/pyccel/pull/732/files#diff-68eb50666798e9569a48a77a7d59e728348769aeea74f48daf6a5b955586170aR31
https://github.com/pyccel/pyccel/pull/732/files#diff-2e950ccaf32b877cb52a5097e9f89778194c0fc725489295c13058f341373016R412
https://github.com/pyccel/pyccel/pull/732/files#diff-2e950ccaf32b877cb52a5097e9f89778194c0fc725489295c13058f341373016R452

Reply:
yes I'm looking thanks